standard_forms/inbound/latest/src/header.html

```diff
@@ -1,48 +1,65 @@
-{% macro dateOrNA(datetime) %}
-{% if datetime %}{{ datetime | date(format="%d/%m/%Y", timezone=arguments.timezone) }}{% else
-%}{{t(k="messages.not-applicable", f="N/A")}}{% endif %}
-{% endmacro input %}
+{% macro dateOrNA(datetime) %} {% if datetime %}{{ datetime |
+date(format="%d/%m/%Y", timezone=arguments.timezone) }}{% else
+%}{{t(k="messages.not-applicable", f="N/A")}}{% endif %} {% endmacro input %}
 
-<table class="header_image_section" style="width: 100%;height: 98%;">
-    <tr>
-        <th style="text-align:left; width:10%;height:70%;">
-            <img class="logo" src="{{data.store.logo}}" />
-        </th>
-        <th style="text-align:left;width:20%">
-            <span style="font-size: 12pt; font-weight: bold;">{{ data.store.storeName }}</span><br>
-            <span style="font-size: 10pt;">{{ data.store.name.address1 }}</span><br>
-            <span style="font-size: 10pt;">{{ data.store.name.address2 }}</span><br>
-            <span style="font-size:6pt;">
-                {{t(k="label.phone",f="Telephone")}}: {{ data.store.name.phone }}
-            </span><br>
-            <span style="font-size:6pt;">
-                {{t(k="label.email",f="Email")}}: {{ data.store.name.email }}
-            </span>
-        </th>
-        <th style="text-align:right;width:70%;font-size:28pt;">{{ t(k="report.inbound-form", f="Inbound Shipment Form")
-            }}</th>
-    </tr>
+<table class="header_image_section" style="width: 100%; height: 98%">
+  <tr>
+    {% if data.store.logo %}
+    <th style="text-align: left; width: 10%; height: 70%">
+      <img class="logo" src="{{data.store.logo}}" />
+    </th>
+    {% endif %}
```

Comment:
```suggestion
    <th style="text-align: left; width: 10%; height: 70%">
        {% if data.store.logo %}
      <img class="logo" src="{{data.store.logo}}" />
          {% endif %}
    </th>
```

Why not this?

Reply:
I'm guessing this is ok as we're not using any table rows here, I take it back...